docs/MAINTAINING.md

```diff
@@ -44,3 +44,80 @@ do?**
 A: We are always open for suggestions on how to make the process for
 contributors or maintainers smooth. If you have any feedback, please open a
 GitHub ticket for discussion.
+
+## Nominating new maintainers
+
+The nomination process is open to all current maintainers. Nominations are
+discussed in private, using
+[GitHub Team Discussions](https://docs.github.com/en/github/setting-up-and-managing-organizations-and-teams/about-team-discussions).
+(Don't confuse Team Discussions with recently introduced project-level
+[Discussions](https://docs.github.com/en/discussions).)
+
+1. To nominate a new maintainer, post a new comment to the Team Discussion of
+   the team you want the new maintainer to join. If you are not sure which team
+   to choose, then pick
+   [loopback-maintainers](https://github.com/orgs/strongloop/teams/loopback-maintainers).
+
+   _New to Team Discussions? Learn more in GitHub docs:
+   [Creating a new discussion](https://docs.github.com/en/github/setting-up-and-managing-organizations-and-teams/creating-a-team-discussion)._
+
+2. Set the title to `Nominate {@handle}`, e.g. `Nominate @bajtos`.
+
+3. In the description, fill in the following template. Use hyperlinks where
+   appropriate, to make it easier for other maintainers to find and review the
+   supporting documentation.
+
+   ```md
+   **GitHub id:** <!-- e.g. @bajtos -->
+
+   **Active github repo(s):**
+
+   <!-- e.g. https//github.com/strongloop/loopback-datasource-juggler -->
+
+   **Reasons for nomination:**
+
+   <!--
+   A short explanation why you are nominating the person. Have they contributed
+   a significant improvement or a series of small changes? Are they active in
+   our issue tracker and/or discussions? Something else?
+   -->
+
+   **Their work & contributions:**
+
+   <!--
+   Please include links to GitHub where people assessing the nomination can
+   review comments and code posted by the nominee.
+
+   Few example URLs:
+
+     All pull requests created by @nflaig in loopback-next repository:
+     https://github.com/strongloop/loopback-next/pulls/nflaig
+
+     Issues in any of `strongloop` repositories where @mitsos1os commented:
+     https://github.com/issues?utf8=✓&q=org%3Astrongloop+involves%3Amitsos1os
+   -->
+   ```
+
+4. Make sure to change the default visibility from
+   `StrongLoop and IBM API Connect` to `Private`.
+
+5. Post a link to the new topic to the Slack channel
+   [#loopback-maintainers](https://loopbackio.slack.com/archives/GRP782GAZ), to
+   let other maintainers know about the nomination and ask for feedback.
+
+6. Allow at least 7 days for everybody to review the nomination. A nomination is
+   accepted if there are at least 2 other maintainers voting for it and there
+   are no (unresolved) objections.
+
+7. Once the nomination is accepted, ask @dhmlau to reach out to the nominated
```

Comment:
I prefer not to be the only designated person to reach out to nominated maintainer (don't want to be the bottleneck as well) :)
I'm thinking maybe put `either @dhmlau or @raymondfeng`? or whoever nominate can reach out to the person directly and maybe just cc either of us?

Reply:
Makes sense, I'll change this step.